datalad/interface/common_cfg.py

```diff
@@ -307,6 +307,13 @@
                'text': 'Git-annex large files expression (see https://git-annex.branchable.com/tips/largefiles; given expression will be wrapped in parentheses)'}),
         'default': 'anything',
     },
+    'datalad.runtime.max-annex-jobs': {
```

Comment:
I think it should be `datalad.annex.jobs` because it is annex specific setting, and pretty much we just pass it into annex call, similarly to the `retry`, whenever `runtime` is more about our own behavior:
```
$> git grep datalad.annex datalad/interface/common_cfg.py
datalad/interface/common_cfg.py:    'datalad.annex.retry': {

$> git grep datalad.runtime datalad/interface/common_cfg.py
datalad/interface/common_cfg.py:    'datalad.runtime.raiseonerror': {
datalad/interface/common_cfg.py:    'datalad.runtime.report-status': {
```